contrib/postgresql/README.md

```diff
@@ -95,12 +93,32 @@ Please make sure that your database user (in this collector) has the rights to c
 
 Function description:
 ---------------------
-The function collectd_insert() creates all tables and columns by itself.
-1. The instance table consists of host/plugin/plugin_inst
-2. The plugin_$plugin table (e.g. plugin_apache) contain all data for that plugin. The function collectd_insert() inserts the value into the column that its type/type_inst/name determines. There is one sad thing about collectd. The times that are submitted dont match 100%, so there is a epsilon (0.5 sec) that is used to check to what row a value belongs. If the column is not yet present it is added by this function.
+There is one sad thing about collectd. The times that are submitted dont match 100%, so there is an epsilon (0.5 sec) that is used to check to what row a value belongs.
+1. The procedure collectd_insert() inserts the values into the incoming table and realign the timestamps. It also creates the instances in the instance table.
+2. The collected data gets moved from the incoming table to the destination tables by the procedure move_data_to_table().
 
 The function impl_location() removes some data anomalies that are there when the data get submitted. There is a default that matches most cases. The plugins cpufreq, ping and memory get their names, plugin_inst get adjusted.
 
+The plugin_$plugin table (e.g. plugin_apache) then contain all data for that plugin.  If the column is not yet present it is added by this function.
+The procedure move_data_to_table() must be called periodically. In my case by a cron job.
+
+```
+root@www-collectd:/etc/cron.d# cat collectd
+# Example of job definition:
+# .---------------- minute (0 - 59)
+# |  .------------- hour (0 - 23)
+# |  |  .---------- day of month (1 - 31)
+# |  |  |  .------- month (1 - 12) OR jan,feb,mar,apr ...
+# |  |  |  |  .---- day of week (0 - 6) (Sunday=0 or 7) OR sun,mon,tue,wed,thu,fri,sat
+# |  |  |  |  |
+# *  *  *  *  * user-name command to be executed
+  0  0  1  *  * root      /opt/collectd/etc/clean-tables.sh
+*/1  *  *  *  * root      /opt/collectd/etc/move-data.sh
+```
+
+The procedure collectd_cleanup() is the maintenance function. It has as an argument the number of days where to keep the data. It can be called by pgagent or a similar mechanism like "CALL collectd_cleanup(180)". This delete all data that is older than 180 days.
```

Comment:
```suggestion
The procedure collectd_cleanup() is the maintenance function. It has as an argument the number of days to keep the data. It can be called by pgagent or a similar mechanism like "CALL collectd_cleanup(180)". This deletes all data that is older than 180 days.
```